src/base/torrentfilter.h

```diff
@@ -60,6 +60,7 @@ class TorrentFilter
 
     static const QString AnyCategory;
     static const QStringSet AnyHash;
+    static const QString TagFilteringDisabled;
```

Comment:
I switched it back to AnyTag and added a comment explaining that "any" means any permutation rather than any individual item.